### PR TITLE
Make PocketGear great again

### PR DESCRIPTION
**Pull request by izambasiron**
Fix errors on ios & android

### CODE REVIEW

File: android/app/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+POKEDB_RELEASE_STORE_FILE=POKEDB_RELEASE_STORE_FILE
```
Comment (satya164) [inline android/app/gradle.properties]: This file is unnecessary

File: android/app/src/main/java/com/wibblystuff/gear/MainActivity.java [hunk 2]
```diff
@@ -24,6 +24,6 @@ protected void onCreate(Bundle savedInstanceState) {
      */
     @Override
     protected String getMainComponentName() {
-        return "main";
+        return "PocketGear";
```
Comment (satya164) [inline android/app/src/main/java/com/wibblystuff/gear/MainActivity.java]: I'd like to keep it as `main` and change `PockerGear` to `main` in `AppDelegate.m`

File: src/components/CPCalculator.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ const styles = StyleSheet.create({
   },
 
   amount: {
-    fontFamily: 'MontserratBold',
+    fontFamily: 'Montserrat-SemiBold',
```
Comment (satya164) [inline src/components/CPCalculator.js]: This will break in Android

File: src/components/PokemonDetails.js [hunk 4]
```diff
@@ -90,15 +89,15 @@ const styles = StyleSheet.create({
 
 
 type Props = {
-  pokemon: Pokemon;
+  screenProps: Object;
```
Comment (satya164) [inline src/components/PokemonDetails.js]: I took a different approach for this

File: android/app/gradle.properties (hunk 1 above)
Comment (izambasiron) [inline android/app/gradle.properties]: Will fail to build, without

File: src/components/CPCalculator.js (hunk 3 above)
Comment (izambasiron) [inline src/components/CPCalculator.js]: I was managed to build

File: src/components/PokemonDetails.js (hunk 4 above)
Comment (izambasiron) [inline src/components/PokemonDetails.js]: I would love to see it, do share

**Comment by satya164** [comment]
I just updated master. Android works fine. Maybe you can help fix the iOS version :)

Thanks a lot!

**Comment by izambasiron** [comment]
Ok, checking it out.

File: src/components/CPCalculator.js (hunk 3 above)
Comment (satya164) [inline src/components/CPCalculator.js]: Build will work, but the font won't work

File: android/app/gradle.properties (hunk 1 above)
Comment (satya164) [inline android/app/gradle.properties]: You can put it in your global gradle directory. I should at it to README

File: src/components/PokemonDetails.js (hunk 4 above)
Comment (satya164) [inline src/components/PokemonDetails.js]: It's in master

**Comment by izambasiron** [comment]
Closing this. Android version works. Fixing ios version.

**Pull request closed without merging**